node/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "azure-pipelines-task-lib",
-  "version": "2.7.8",
+  "name": "vsts-task-lib",
```

Comment:
did you mean to change the name?

Reply:
Whoops no, not sure how that happened - good catch.